backend/eventloop/application_event_loop/application_event_runner_deployments_test.go

```diff
@@ -291,3 +291,203 @@ var _ = Describe("Application Event Runner Deployments to check SyncPolicy.SyncO
 
 	})
 })
+
```

Comment:
This test will be removed/kept based on reviews whether backend needs this test or not